src/Fabulous.Avalonia/Widgets/Controls/ProgressBar.fs

```diff
@@ -0,0 +1,77 @@
+namespace Fabulous.Avalonia
+
+open Avalonia.Controls
+open Avalonia.Layout
+open System.Runtime.CompilerServices
+
+open Fabulous
+
+type IFabProgressBar =
```

Comment:
You have another PR with the progress bar :). Happy to take this PR with both controls. Just make sure it has all the ProgressBar modifiers

Reply:
Ah, sorry, I mistakenly included this progress bar in this branch also. I think once the other pull request is merged that this will resolve without conflict. I'll add those repeat button samples shortly...